### PR TITLE
Update getting started guide

### PR DESCRIPTION
**Pull request by lord**
- Bumps Ruby version to match Gemfile
- Moves around some things to be (hopefully) easier to understand
- Adds better instructions for installing Java if needed, and some other gotchas

:heart:, doc-a-thon

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +104,8 @@ $ cd client
 $ lein cljsbuild auto
 ```
 
+Lein can take as long as ~30 seconds to build all the Clojurescript, so be patient! This site won't work it's done. Once it's finished, you can visit the site at [localhost:5001](http://localhost:5001/).
```
Comment (zachallaun) [inline README.md]: Typo: `This site won't work it's done.`

**Comment by zachallaun** [comment]
Thanks for the readme improvements! I made a note about a typo, and when that's fixed I'll happily merge.

**Comment by lord** [comment]
@zachallaun whoops, good catch, sorry about that! Should be better now, LMK if you'd rather this was squashed.

**Comment by zachallaun** [comment]
Thanks for the fix, @lord! Don't worry about squashing. Merging this in.

**Pull request merged**